pom.xml

```diff
@@ -162,6 +162,10 @@
                         <Private-Package>
                             org.glassfish.gmbal.util, org.glassfish.gmbal.impl, org.glassfish.gmbal.typelib
                         </Private-Package>
+                        <Import-Package>
+                            org.glassfish.pfl.*;version="[\$(version;=-;\$(@)),\$(version;+;\$(@)))",
```

Comment:
The version of pfl jumped to 4.1, so by default, [4.1,5) is being generated but what we want is to have [4.0, 5) being imported - assuming that there is no API change between PFL 4.0.1 and 4.1.0. If there is then this should be removed

Reply:
There were a number of API changes. Unused methods were removed, which should not affect anything, but in addition, some methods were added to the Wrapper and Bridge classes to support JDK11.

Reply:
OK, removed this part